src/pages/Payment/Payment.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import './Payment.scss';
+
+const Payment = () => {
+  const [inputs, setInputs] = useState({
+    lastName: '',
+    firstName: '',
+    address: '',
+    phoneNumber: Number(''),
```

Comment:
```suggestion
    phoneNumber: 0,
```
0으로 작성하면 number type의 초기값을 줄 수 있습니다.